source/spec/vagrant-openstack-provider/config_resolver_spec.rb

```diff
@@ -506,42 +520,66 @@
       context 'with string volume id' do
         it 'returns normalized volume' do
           config.stub(:volume_boot) { '001' }
-          expect(@action.resolve_volume_boot(env)).to eq id: '001', device: 'vda'
+          nova.stub(:get_all_images).with(anything) do
```

Comment:
When booting from an existing volume the method `get_all_images` should never be called. Thus, this stub should not be necessary. If the method is calls it is probably a bug.